plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray.yaml

```diff
@@ -0,0 +1,162 @@
+            # For more documentation on available fields, see:
+            # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
+            head_node:
+                InstanceType: m5.large
+                ImageId: ami-008d8ed4bd7dc2485 # Deep Learning AMI (Ubuntu 18.04) Version 27.0
+
+                # You can provision additional disk space with a conf as follows
+                BlockDeviceMappings:
+                    - DeviceName: /dev/sda1
+                      Ebs:
+                          VolumeSize: 100
+
+                # Additional options in the boto docs.
+
+            # Provider-specific config for worker nodes, e.g. instance type. By default
+            # Ray will auto-configure unspecified fields such as SubnetId and KeyName.
+            # For more documentation on available fields, see:
+            # http://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.ServiceResource.create_instances
+            worker_nodes:
+                InstanceType: m5.large
+                ImageId: ami-008d8ed4bd7dc2485 # Deep Learning AMI (Ubuntu 18.04) Version 27.0
+
+                # Run workers on spot by default. Comment this out to use on-demand.
+                InstanceMarketOptions:
+                    MarketType: spot
+                    # Additional options can be found in the boto docs, e.g.
+                    #   SpotOptions:
+                    #       MaxPrice: MAX_HOURLY_PRICE
+
+                # Additional options in the boto docs.
+
+            # Files or directories to copy to the head and worker nodes. The format is a
+            # dictionary from REMOTE_PATH: LOCAL_PATH, e.g.
+            file_mounts: {
+            #    "/path1/on/remote/machine": "/path1/on/local/machine",
+            #    "/path2/on/remote/machine": "/path2/on/local/machine",
+            }
+
+            # List of commands that will be run before `setup_commands`. If docker is
+            # enabled, these commands will run outside the container and before docker
+            # is setup.
+            initialization_commands: []
+
+            # List of shell commands to run to set up nodes.
+            setup_commands:
+                # Note: if you're developing Ray, you probably want to create an AMI that
+                # has your Ray repo pre-cloned. Then, you can replace the pip installs
+                # below with a git checkout <your_sha> (and possibly a recompile).
+                - conda create -n hydra37 python=3.7 -y # Ray is only officially supported up to python 3.7
+                - echo 'export PATH="$HOME/anaconda3/envs/hydra37/bin:$PATH"' >> ~/.bashrc
+                - pip install -U ray
+                - pip install -U https://hydra-test-us-west-2.s3-us-west-2.amazonaws.com/hydra_core-1.0.0rc1-py3-none-any.whl
+                - pip install -U https://hydra-test-us-west-2.s3-us-west-2.amazonaws.com/hydra_ray_launcher-0.1.0-py3-none-any.whl
```

Comment:
1. put a todo here, we will want to clean this up.
2. also, it would be good to start publishing nightly timestamped builds automatically to allow people to more easily install the latest version from master.